crates/typed-store/src/rocks/mod.rs

```diff
@@ -1183,7 +1183,28 @@ impl<K, V> DBMap<K, V> {
         )
     }
 
-    // Creates an RocksDB read option with lower and upper bounds set corresponding to `range`.
+    // Creates a RocksDB read option with specified lower and upper bounds.
```

Comment:
Mention if the lower / upper bound is inclusive / exclusive?

Reply:
Updated